e2e/setup.go

```diff
@@ -586,21 +592,22 @@ func (cfg SystemConfig) Start(_opts ...SystemConfigOption) (*System, error) {
 		}
 	}
 
-	// Validator (L2 Output Submitter + Challenger)
+	// Run validator node (L2 Output Submitter, Asserter)
 	validatorCliCfg := validator.CLIConfig{
 		L1EthRpc:                     sys.Nodes["l1"].WSEndpoint(),
 		RollupRpc:                    sys.RollupNodes["proposer"].HTTPEndpoint(),
 		L2OOAddress:                  predeploys.DevL2OutputOracleAddr.String(),
 		ColosseumAddress:             predeploys.DevColosseumAddr.String(),
 		ValPoolAddress:               predeploys.DevValidatorPoolAddr.String(),
-		SecurityCouncilAddress:       predeploys.DevSecurityCouncilAddr.String(),
-		ChallengerPollInterval:       50 * time.Millisecond,
-		OutputSubmitterRetryInterval: 50 * time.Millisecond,
-		OutputSubmitterRoundBuffer:   30,
+		ChallengerPollInterval:       500 * time.Millisecond,
```

Comment:
Why did you change?

Reply:
 I don't see a need for settings with too short a cycle.
Since we don't inject all the timing variables used in the code at test time, lowering this value by a lot will not make the test time any shorter. Rather, it will cause a console load when debugging.